### DISCUSSION TITLE
API Endpoint to List Radius Groups

### DISCUSSION THREAD

**Discussion by 3pleo** [openwisp/openwisp-radius#552] [Q&A] [answered]
Hi,

I can't seem to find any API endpoint to list the configured Radius Groups (/admin/openwisp_radius/radiusgroup/) in openwisp api/v1/docs/

How can I achieve this?

Thanks.
Regards.

**Comment by nemesifier** [comment, marked as answer]
At the moment there's no such API endpoint, we need to add one.

**Reply by nemesifier** [reply]
PS: https://github.com/openwisp/openwisp-radius/issues/553
